src/styles/card.scss

```
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    max-width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin: 40px 0 4px;
+
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+
+    font-weight: $font-weight-400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secon-main-color;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__stars {
+    display: flex;
+    align-content: center;
+  }
+
+  &__reviews {
+    font-weight: $font-weight-400;
+    font-size: 10px;
+    color: $main-text-color;
+    text-align: center;
+  }
+
+  &__bottom {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &_text {
+      font-weight: $font-weight-400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $secon-main-color;
+    }
+
+    &_price {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
+
+  &__button {
+    height: 40px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
```

create variable for all colors
```suggestion
    color: $c-white;
```